lib/presentation/di/module/app_module.dart

```diff
@@ -332,6 +332,9 @@ class AppModule {
       getIt<AuthenticationRepository>(),
       getIt<TokenRepository>(),
       getIt<CredentialRepository>()));
+    getIt.registerFactory(() => RemovePermanentTokenInteractor(
+        getIt<TokenRepository>(),
+        getIt<CredentialRepository>()));
```

Comment:
what is the different with DeletePermanentToken?

Reply:
`RemovePermanentToken` doesnt delete the token from server side to prevent account from getting locked

Reply:
still not understand